app/models/concerns/wakes/metrics/google_analytics_pageviews.rb

```diff
@@ -11,6 +11,14 @@ def enqueue_pageview_count_update
           Wakes::GoogleAnalyticsPageviewJob.perform_later(self)
         end
 
+        def google_analytics_profile_id
+          if host.empty? || (host == ENV['DEFAULT_HOST'])
```

Comment:
@chrismaddocks we tend to prefer `blank?` over `empty?`. See http://stackoverflow.com/questions/885414/a-concise-explanation-of-nil-v-empty-v-blank-in-ruby-on-rails